javascript/workflow/src/verbs/print.ts

```diff
@@ -0,0 +1,21 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import type { PrintArgs } from '@datashaper/schema'
+
+import type { ColumnTableStep } from './util/factories.js'
+import { stepVerbFactory } from './util/factories.js'
+
+export const DEFAULT_LIMIT_ROWS = 10
```

Comment:
You can put this in workflow/src/verbs/defaults so all default verb args are centralized